drivers/ethernet/eth_mcux.c

```diff
@@ -1059,6 +1073,16 @@ static void eth_mcux_init(const struct device *dev)
 			kENET_RxAccelProtoCheckEnabled;
 	}
 
+#ifdef CONFIG_ETH_MCUX_RMII_EXT_CLK
+	if(context->base == ENET) {
+		IOMUXC_EnableMode(IOMUXC_GPR, kIOMUXC_GPR_ENET1RefClkMode, true);
+		IOMUXC_EnableMode(IOMUXC_GPR, kIOMUXC_GPR_ENET1TxClkOutputDir, false);
+	} else {
+		IOMUXC_EnableMode(IOMUXC_GPR, kIOMUXC_GPR_ENET2RefClkMode, true);
```

Comment:
I'm not sure this is required- I think we enable these clocks in the RT10xx SOC file: https://github.com/zephyrproject-rtos/zephyr/blob/main/soc/arm/nxp_imx/rt/soc_rt10xx.c#L201

Reply:
I've just seen this this was merged to `main` a few days ago. I've pulled it in, and updated it on this PR to support both external refclk, and the rt1064 generating the refclk.